ci/unit/test_main_node.sh

```diff
@@ -15,6 +15,16 @@ sudo mount -t tmpfs -o size=16G tmpfs /mnt/daos
 sudo mkdir -p "$DAOS_BASE"
 sudo mount -t nfs "$HOSTNAME":"$HOSTPWD" "$DAOS_BASE"
 sudo cp "$DAOS_BASE/install/bin/daos_admin" /usr/bin/daos_admin
+set +x
+if [ -n "$BULLSEYE" ]; then
+  pushd "$DAOS_BASE/bullseye"
+    sudo ./install --quiet --key "${BULLSEYE}" \
+                   --prefix /opt/BullseyeCoverage
```

Comment:
Can this be installed in a local dir, without `sudo`?  Doing so would considerably lower the security risk of running this `./install`.

Reply:
It can be done, but it would take some investigation on how to get scons to find the new location.
It looks like a lot of things just assume that bulleye is going to reside at that path.